src/controllers/wasteDisposalAndRecoveryCodes.controller.js

```diff
@@ -0,0 +1,10 @@
+'use strict'
+
+const BaseController = require('./base.controller')
+const { WASTE_RD_DISPOSAL: { path: wasteDisposalCodesBasePath } } = require('../routes')
+
+module.exports = class WasteDisposalAndRecoveryCodesController extends BaseController {
+  async doGet (request, h) {
+    return this.redirect({ h, path: `${wasteDisposalCodesBasePath}/0` })
```

Comment:
Is this controller a place-holder for some coming functionality?

Reply:
No it's just the simplest possible handler to act as an entry point for the "do the disposal and recovery codes" task - it just redirects to the first activity.